### PR TITLE
Compatibility with gtk (crate) 0.14.0

### LINKED ISSUE #2: Compatibility with gtk (crate) 0.14.0
Hi :) 
I am interested in having a version of gladis that is compatible with [gtk 0.14.0](https://crates.io/crates/gtk/0.14.0). There have been quite a few changes, but the only change that affects gladis would be the renaming of `get_object` on builder to [`object`](https://gtk-rs.org/gtk3-rs/stable/latest/docs/gtk/prelude/trait.BuilderExtManual.html#tymethod.object). 
Can I submit a PR with this change? I am not sure how you would like to handle this, as it would be a breaking change!

### PR DESCRIPTION
**Pull request by xou816**
Fixes #2 

(I added a commit with the version bump, but I can remove it if there are other tasks to complete before releasing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,7 @@ impl Window {
 }
 ```
 
-Without Gladis, you would have to manually parse each of the Glade entries like
-described in the [official Gtk-rs Glade
-tutorial](https://gtk-rs.org/docs-src/tutorial/glade):
+Without Gladis, you would have to manually parse each of the Glade entries.
```
Comment (xou816) [inline README.md:35]: (couldn't find any updated docs for Glade usage on the gtk-rs website :/ there's always https://gtk-rs.org/gtk3-rs/stable/latest/docs/gtk/struct.Builder.html but it references C functions...)

Comment (gagath) [reply README.md:35]: Fine.

**Review by gagath — CHANGES_REQUESTED** [review]
Please remove the v1.0.0 version bump part. Keep the changes described in the "Unreleased" section of the changelog. I will release a new version after this PR is merged.

**Comment by xou816** [comment]
Done! :)

**Comment by xou816** [comment]
I have 85cf17f51bd42b3f964769ef763c3cb102091c22 on my copy of master, but I don't see it here anymore, should I remove it?

**Comment by gagath** [comment]
> I have [85cf17f](https://github.com/MicroJoe/gladis/commit/85cf17f51bd42b3f964769ef763c3cb102091c22) on my copy of master, but I don't see it here anymore, should I remove it?

It was my mistake, I have updated master, can you rebase the PR?

**Comment by gagath** [comment]
Rebased, merged, and published in `v1.0.0` ([Github release](https://github.com/MicroJoe/gladis/releases/tag/v1.0.0), [crates.io](https://crates.io/crates/gladis)), thanks!

**Comment by xou816** [comment]
I am bit late for the rebase but I see you handled it :) thank you!!

**Pull request merged**